content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -635,6 +635,14 @@ or updating objects that contain Pod templates, such as Deployments, Jobs, State
 See [Enforcing Pod Security at the Namespace Level](/docs/concepts/security/pod-security-admission)
 for more information.
 
+### rbac.authorization.kubernetes.io/autoupdate
+
+Examples: `rbac.authorization.kubernetes.io/autoupdate: false`
+
+Used on: ClusterRole, ClusterRoleBinding
+
+When this annotation is set to `true`, default RBAC ClusterRole and ClusterRoleBinding objects are automatically updated at server start to add missing permissions and subjects (extra permissions and subjects are left in place). To prevent autoupdating a particular role or rolebinding, set this annotation to `false`.
```

Comment:
If I make my own ClusterRole and set this annotation, what happens?

Reply:
`kubectl auth reconcile` (which allows reconciling arbitrary RBAC objects in a manifest file) will respect the annotation